### ISSUE TITLE
Some questions about the scripts

### ISSUE THREAD

**Issue by zengkaipeng** [yuqianghan/editretro#1]
What''s the `sort` in generate.sh for? Are the results ranked according to the predicted logits when calculating the top-k accuracy?

**Comment by yuqianghan** [comment]
In Fairseq, the output generations from the fairseq-generate command are not ordered in the same way as the input. The sort can be used to reorder the generated outputs to match the original input order.
